filesystems/blob/src/test/resources/mockito-extensions/org.mockito.plugins.MockMaker

```diff
@@ -0,0 +1 @@
+mock-maker-inline
```

Comment:
What is this used for?

Reply:
This is required to mock final classes, which we need to mock the AzureFileSystem class. Particularly with the move of the expiration information onto the AzureFileSystem class, the different token refresh paths could only be tested with a mock of the `isExpired` method